features/allow_rescue.feature

```diff
@@ -33,7 +33,7 @@ Feature: Allow Cucumber to rescue exceptions
       end
 
       Then('I should see the public error page') do
-        expect(page).to have_content "We're sorry, but something went wrong."
```

Comment:
Question: I get that this change is needed, but not why. Is this something to do with the way rails renders things. Is there a doc link for this I can see quickly?

Reply:
> but not why. 

They're just simply starting using typographic apostrophes, it's also an "issue" with validations, but they changed their mind

https://github.com/rails/rails/commit/acfa0454058c535c1352ddab91c93ca914f0805a

I will add a ref in the git commit

https://github.com/rails/rails/blame/a4e1ff72fbb24b274fce95507199c5a5f9541898/railties/lib/rails/generators/rails/app/templates/public/500.html#L7

Ref: rails/rails#53045

Reply:
For now I've raised a question to rafael on the rails repo. Want to hear his thoughts before doing this.